http/file_server_test.ts

```diff
@@ -975,42 +975,87 @@ Deno.test(
 
 Deno.test("file_server prints local and network urls", async () => {
   const port = await getAvailablePort();
+  const process = spawnDeno([
+    "--allow-net",
+    "--allow-read",
+    "--allow-sys=networkInterfaces",
+    "http/file_server.ts",
+    "--port",
+    `${port}`,
+  ]);
+  const output = await readUntilMatch(process.stdout, "Network:");
+  const networkAdress = Deno.networkInterfaces().find((i) =>
+    i.family === "IPv4" && !i.address.startsWith("127")
+  )?.address;
```

Comment:
It seems like this should be used as the implementation for `getNetworkAddress()`.